project/DockerSetup.scala

```diff
@@ -10,12 +10,12 @@ import sbt.TaskKey
 
 object DockerSetup {
   def setupDocker(stage: TaskKey[File], dockerFileMod: Dockerfile => Dockerfile = identity) = {
-    val nameSpacePrefix = Option(System.getProperty("repository")) match {
+    val nameSpacePrefix = Option(System.getProperty("repository", System.getenv("MAGDA_DOCKER_REPOSITORY"))) match {
```

Comment:
Do we need to support `System.getProperty("repository")` as well (support passing value both ways)? 
We still have `docker-build-local` script needs it:
```
"docker-build-local": "cd .. && sbt \"-Drepository=localhost:5000\" searchApi/dockerBuildAndPush",
```

Reply:
Yep - that's what this should do! System.getProperty gets the value passed in via -D, but if it's not present it falls back to the MAGDA_DOCKER_REPOSITORY env var. Is it not working like that for you?

Reply:
Ah~ Sorry I missed that line. All good now 👍